introduction.md

```diff
@@ -1 +1,4 @@
 Hello, my name is Chris, and my favorite Code School path is Electives
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?